ui/__tests__/components/thing-counters.test.jsx

```diff
@@ -0,0 +1,46 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import ThingCounter from '../../components/thing-counters';
+import mockRouter from '../util/mockRouter';
+
+
+describe('<ThingCounter />', () => {
+  it('states zero results when there are no results', () => {
+    const context = { router: mockRouter() };
+    const result = shallow(<ThingCounter count={0} chunk={0} singular={ "requirement" } plural={ "requirements" }/>, { context });
```

Comment:
I _think_ you can just do `singular="requirement"` here. You may also be able to do `count=0` rather than using so many curlies.